.circleci/config.yml

```diff
@@ -12,10 +12,10 @@ jobs:
             - v8-dependency-cache-
       - run:
           name: Install Prereqs
-          command: sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev
+          command: sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev yarn
       - run:
           name: Install Dependencies
-          command: npm install --quiet
+          command: npx lerna bootstrap
```

Comment:
how is `npx lerna` different from `lerna`?

Reply:
Lerna is an npm package, so this is used to avoid having to install it on the machine beforehand. It can't be installed with the rest of the local packages because it is being _used_ to install the rest of the packages :).